<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>GUI</string>
       </attribute>
       <widget class="QPushButton" name="westButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>590</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>West</string>
        </property>
       </widget>
       <widget class="QPushButton" name="southButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>630</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>South</string>
        </property>
       </widget>
       <widget class="QPushButton" name="northButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>550</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>North</string>
        </property>
       </widget>
       <widget class="QPushButton" name="eastButton">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>590</y>
          <width>51</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>East</string>
        </property>
       </widget>
       <widget class="QPushButton" name="upButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>550</y>
          <width>80</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Upstairs</string>
        </property>
       </widget>
       <widget class="QPushButton" name="downButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>620</y>
          <width>80</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Downstairs</string>
        </property>
       </widget>
       <widget class="SharedTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>0</y>
          <width>471</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="mapButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>600</y>
          <width>80</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Map</string>
        </property>
       </widget>
       <widget class="QLabel" name="mapLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1251</width>
          <height>531</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <zorder>westButton</zorder>
       <zorder>southButton</zorder>
       <zorder>eastButton</zorder>
       <zorder>upButton</zorder>
       <zorder>downButton</zorder>
       <zorder>textBrowser_2</zorder>
       <zorder>mapButton</zorder>
       <zorder>mapLabel</zorder>
       <zorder>northButton</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Text Based</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>331</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>131</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter Command</string>
        </property>
       </widget>
       <widget class="SharedTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>1211</width>
          <height>551</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>10</y>
          <width>80</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
       <widget class="QLabel" name="mapLabel2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>1211</width>
          <height>641</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <widget class="QLabel" name="mapLabel3">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>1</y>
          <width>1211</width>
          <height>691</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SharedTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>sharedtextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
